crypto/ec/ec_lib.c

```diff
@@ -1559,6 +1566,9 @@ EC_GROUP *EC_GROUP_new_from_params(const OSSL_PARAM params[],
         }
         return group;
     }
+#ifdef FIPS_MODULE
+    return NULL;
```

Comment:
Should we raise an error here about unsupported explicit parameters?

Reply:
Good point, I'll add it.